src/v2/ion/i18nTransformer.js

```diff
@@ -45,11 +45,17 @@ const I18N_OVERRIDE_MAPPINGS = {
   'identify.credentials.passcode': 'primaryauth.password.placeholder',
   'identify.rememberMe': 'remember',
 
-  'select-authenticator-enroll.authenticator.email': 'oie.email',
-  'select-authenticator-enroll.authenticator.password': 'oie.password',
-  'select-authenticator-enroll.authenticator.phone': 'oie.phone',
+  'select-authenticator-enroll.authenticator.email': 'oie.authenticator.email.label',
+  'select-authenticator-enroll.authenticator.password': 'oie.authenticator.password.label',
+  'select-authenticator-enroll.authenticator.phone': 'oie.authenticator.phone.label',
   'select-authenticator-enroll.authenticator.security_key': 'oie.webauthn',
```

Comment:
change format for keys `oie.webauthn` and `oie.app` as well.

Reply:
@anipendakur-okta  can you please update the oie.app to oie.okta.verify please. Thanks! I dont want to create a new pr for this.